lib/utils/fsUtils.ts

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+import { promisify } from 'util';
+
+// Promisified wrappers for file system methods - only required for support for NodeJS v9 and older
+// NodeJS v10 provides promisified file system methods out of the box
```

Comment:
We could switch to `>= 10.13.0 LTS` and not have these wrappers.  Is there a good reason to support lower versions?

Reply:
We could, I just wasn't sure whether we should drop support for it yet. For now, I think supporting Node 8 makes sense because it is still "Active" according to the [release chart](https://github.com/nodejs/Release). Then again, by the time xud is production-ready I'd expect that Node 8 will be on its way out. Longer term I think it makes sense to try to support every version that's still being maintained.

Thoughts @moshababo?

Reply:
Yes, i'd say let's not bound ourself yet to support only v10.

Reply:
Sounds good. :).